### DISCUSSION TITLE
Installation Homematic Integration nicht möglich / Verbindung fehlgeschlagen

### DISCUSSION THREAD

**Discussion by Domme86** [SukramJ/aiohomematic#1290] [Q&A]
Hallo,

Home Assistant wie die RaspberryMatic auf Home Assistant laufen fehlerfrei.
Ich bekomme die Integration, welche ebenfalls fehlerfrei installiert wurde nicht zum Laufen.
Die Firewall ist offen wie ein Scheunentor. Einen neuen Nutzer mit/ohne Adminrechten habe ich angelegt.

Ich erhalte folgende Fehlermeldungen im Protokoll:
Can't resolve host for de838cd8_raspberrymatic
OSError on temporary_instance-BidCos-RF: (-5, 'Name has no usable address')
OSError on temporary_instance-HmIP-RF: (-5, 'Name has no usable address')

**Comment by SukramJ** [comment]
Die Fehlermeldung  `Can't resolve host for de838cd8_raspberrymatic` ist ziemlich eindeutig: Der Name `de838cd8_raspberrymatic` kann nicht in eine IP Adresse aufgelöst werden.
Das ist ein Problem auf der Maschine wo HA installiert ist, aber kein Problem der Integration.
Ich nehme mal an Du versuchst von HA auf das Raspberrymatic-Addon zuzugreifen?

**Reply by Domme86** [reply]
Korrekte Annahme. Hast du einen Tip zur Problemlösung?

**Reply by SukramJ** [reply]
Eigentlicht nicht, da ich die das Addon nicht verwende. Ich würde mal den kompletten HA-Host, nicht nur HA, neustarten.

**Reply by Domme86** [reply]
Habe ich leider schon gemacht, das Add On und die Integration gelöscht und neu installiert

**Reply by Domme86** [reply]
Und noch ein neuer Fund im Bereich 
Updating Homematic RF-Hardware: no GPIO/USB connected RF-hardware found

RasperryMatic läuft allerdings einwandfrei
